README.md

```diff
@@ -11,6 +11,8 @@
 
 ---
 
+[![Netlify Status](https://api.netlify.com/api/v1/badges/cd5d6330-1628-4d0e-aac1-84f8cf4b7863/deploy-status)](https://app.netlify.com/sites/reactjs-ph-website/deploys)
+
```

Comment:
Just wondering if you intended for it to be on a separate line than the other badges:

![image](https://user-images.githubusercontent.com/7394331/80931498-c4191d80-8dec-11ea-817e-4e852e4c621d.png)

Reply:
Yup, it was intended.